lib/streamlit/elements/slider.py

```diff
@@ -102,6 +104,9 @@ def slider(
             An optional tuple of args to pass to the callback.
         kwargs : dict
             An optional dict of kwargs to pass to the callback.
+        disabled : bool
+            An optional boolean, which disables the slider if set to True. The
+            default is False. This is a keyword only argument.
```

Comment:
nit: The meaning of `This is a keyword only argument` might not be super clear for all users. Maybe we can find a slightly more helpful explanation, e.g.: `This argument can only be supplied by keyword.`